third-party/realdds/include/realdds/topics/dds-topic-names.h

```diff
@@ -21,5 +24,179 @@ constexpr char const * NOTIFICATION_TOPIC_NAME = "/notification";
 constexpr char const * CONTROL_TOPIC_NAME = "/control";
 constexpr char const * METADATA_TOPIC_NAME = "/metadata";
 
+
```

Comment:
Now this file declares both the topic names and their structure. Maybe `dds-topics.h` is a better name?

Reply:
So you want to rename `realdds/topics/dds-topic-names.h` to `realdds/topics/dds-topics.h`? I kinda like the 'names' part...

Reply:
OK